code/modules/mob/living/carbon/human/species.dm

```diff
@@ -399,7 +398,10 @@
 				else
 					hair_overlay.color = forced_colour
 				hair_overlay.alpha = hair_alpha
-				hair_overlay.pixel_y += hair_y_offset
+				if("face" in H.dna.species.offset_features)
+					var/list/offset_list = H.dna.species.offset_features["face"]
+					hair_overlay.pixel_x += offset_list[1]
+					hair_overlay.pixel_y += offset_list[2]
```

Comment:
No need for another list in any of these, just do
`hair_overlay.pixel_y += H.dna.species.offset_features["face"][2]`

Reply:
thanks